firmware/arduino/chessboard/uart.ino

```diff
@@ -84,6 +84,15 @@ bool validateMessageFromPi(volatile char * message)
             return false;
         }
     }
+    else if (message[0] == INSTRUCTION)
+    {
```

Comment:
Please add a comment explaining that you're checking if message[5] is not a valid instruction type